rmf_fleet_adapter_python/src/adapter.cpp

```diff
@@ -331,23 +332,38 @@ PYBIND11_MODULE(rmf_adapter, m) {
     double recharge_threshold,
     double recharge_soc,
     bool account_for_battery_drain,
-    const std::string& finishing_request_string = "nothing")
+    const std::string& finishing_request_string = "nothing",
+    std::shared_ptr<rclcpp::Node> node = nullptr)
     {
+      std::function<rmf_traffic::Time()> time_now = nullptr;
+      std::optional<std::string> planner_id = std::nullopt;
+      if (node)
+      {
+        time_now = [n = std::weak_ptr<rclcpp::Node>(node)]()
+        {
+          const auto node = n.lock();
+          if (!node)
+          {
+            return std::chrono::steady_clock::now();
```

Comment:
If the ROS2 node isn't available, `system_clock` is more likely to give a time that makes sense, so I'd suggest using that instead.

Reply:
gotcha, I did a tiny workaround to lump `system_clock` to `rmf_traffic::Time` which is based on `steady_clock`. Was there any specific reason we were using `steady_clock` for `rmf_traffic::Time` in the past instead of system?
https://github.com/open-rmf/rmf_ros2/pull/267/commits/6815ea70e6fd9ec4fa03c82523c2c134c1aa9f50